packages/expo-background-fetch/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
 
 - Migrated from `@unimodules/core` to `expo-modules-core`. ([#13757](https://github.com/expo/expo/pull/13757) by [@tsapeta](https://github.com/tsapeta))
 
+## 9.2.1 — 2021-08-02
```

Comment:
can you move this under bug fixes for unpublished? we handle moving the entries under the correct version when we publish

Reply:
Changes made. :)

Thank you!